mkt/settings.py

```diff
@@ -381,7 +381,7 @@
 # The verification URL, the addon id will be appended to this. This will
 # have to be altered to the right domain for each server, eg:
 # https://receiptcheck.addons.mozilla.org/verify/
-WEBAPPS_RECEIPT_URL = '/verify/'
+WEBAPPS_RECEIPT_URL = 'http://mp.dev/receipt-verifier/'
```

Comment:
You are right, static_url prepends SITE_URL on to it. Although its set to a full URL on prod, we should just make it relative and let SITE_URL fix it up.